controllers/tasksController.js

```diff
@@ -0,0 +1,66 @@
+const tasks = require('../models/tasks')
+/**
+ * Creates new task
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Task object
+ * @param res {Object} - Express response object
+ */
+const addNewTask = async (req, res) => {
+  try {
+    const task = await tasks.updateTask(req.body)
+    return res.json({
+      message: 'Task created successfully!',
+      task: task.taskDetails,
+      id: task.taskId
+    })
+  } catch (err) {
+    logger.error(`Error while creating new task: ${err}`)
+    return res.boom.badImplementation('An internal server error occurred')
+  }
+}
+/**
+ * Fetches all the tasks
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const fetchTasks = async (req, res) => {
+  try {
+    const allTasks = await tasks.fetchTasks()
+    return res.json({
+      message: 'Tasks returned successfully!',
+      tasks: allTasks.length > 0 ? allTasks : []
+    })
+  } catch (err) {
+    logger.error(`Error while fetching tasks ${err}`)
+    return res.boom.badImplementation('An internal server error occurred')
+  }
+}
```

Comment:
Are we not returning the error `404` if no tasks are found ? I can see it mentioned in the swagger docs below! Please add it here as well.

![image](https://user-images.githubusercontent.com/66248039/104439856-a032ec00-55b7-11eb-83ab-c501aa7153fc.png)


Thank you!